docs/source/contributing/local-deployment.rst

```diff
@@ -137,21 +125,14 @@ First time configuration
       git clone https://github.com/Substra/substra-frontend.git
 
 
-5. Install substra in editable mode
-
-   .. code-block:: bash
-
-      cd substra
-      pip install -e .
-
-6. Install frontend dependencies
+5. Install frontend dependencies
 
    .. code-block:: bash
 
       cd substra-frontend
       npm install --dev
 
-7. Update Helm charts
+6. Update Helm charts
```

Comment:
Add the `cd ..` at the end of each block ? I sometimes finds it confusing when there is "almost" all command to run.

Reply:
hum no strong opinion about the `cd ..`